css/join.css

```diff
@@ -0,0 +1,162 @@
+@import url('https://fonts.googleapis.com/css2?family=Roboto:wght@100;300;400;500;700;900&display=swap');
```

Comment:
Please move this to the HTML document.

Reply:
Can I know the reason for this

Reply:
Please refer to this:
https://web.dev/font-best-practices/